src/lib/Pagination/Mapper/AbstractPagerContentToDataMapper.php

```diff
@@ -88,19 +88,17 @@ protected function isContentIsUser(Content $content): bool
     }
 
     /**
-     * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+     * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
      *
      * @return \eZ\Publish\API\Repository\Values\User\User|null
      */
-    protected function getContributor(ContentInfo $contentInfo): ?User
+    protected function getVersionContributor(VersionInfo $versionInfo): ?User
```

Comment:
Not sure if renaming protected function in abstract class is the safest way to do, it should be internal but maybe someone else have opinion on that too.

Reply:
May I create a separate `getVersionContributor` method, and keep `getContributor` as is? And, mark `getContributor` with `@deprecated`?